libraries/pbrlib/genglsl/mx_dielectric_bsdf.glsl

```diff
@@ -61,20 +54,29 @@ void mx_dielectric_bsdf_transmission(vec3 V, float weight, vec3 tint, float ior,
 
     FresnelData fd;
     if (bsdf.thickness > 0.0)
+    { 
         fd = mx_init_fresnel_dielectric_airy(ior, bsdf.thickness, bsdf.ior);
+    }
     else
+    {
         fd = mx_init_fresnel_dielectric(ior);
-
+    }
     vec3 F = mx_compute_fresnel(NdotV, fd);
 
     vec2 safeAlpha = clamp(roughness, M_FLOAT_EPS, 1.0);
     float avgAlpha = mx_average_alpha(safeAlpha);
+
     float F0 = mx_ior_to_f0(ior);
     vec3 comp = mx_ggx_energy_compensation(NdotV, avgAlpha, F);
     vec3 dirAlbedo = mx_ggx_dir_albedo(NdotV, avgAlpha, F0, 1.0) * comp;
     bsdf.throughput = 1.0 - dirAlbedo * weight;
 
-    bsdf.response = (scatter_mode == 2) ? tint * weight * bsdf.throughput : vec3(0.0);
+    // For now, we approximate the appearance of dielectric transmission as
+    // glossy environment map refraction, ignoring any scene geometry that
+    // might be visible through the surface.
+    fd.refraction = true;
+    vec3 Li = $refractionEnv ? mx_environment_radiance(N, V, X, safeAlpha, distribution, fd) : $refractionColor;
```

Comment:
Out of curiosity why is there an option to replace the refracted environment with a uniform color? We don't have a corresponding control for environment reflections. Or is this just for debugging for now?

Reply:
That option was primarily added for our GLSL/OSL reference renders, since OSL `testrender` refracts the constant-color _background_ instead of the distant environment.

Reply:
Aha! Maybe we should fix this so the environment is visible in both reflection and transmission, for both GLSL and OSL test suite rendering. Lets do that in a separate PR though.